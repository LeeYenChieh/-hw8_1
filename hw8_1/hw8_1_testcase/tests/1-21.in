Q5NIUl9gu22TQL XcXYeZrl5v4MGQWgza4A5pQWZa
CJSbFSXIqcGhdSRqIR EZsBd4XY5c
n2JkZAfCuc61SoWX6jfJaXeWFQbX7VlirzGnEeLrZ YsLJ1vUt63zS9zZ74l
n1KFnP8urpcnEgXNEJs1oN3TAtPEWbgF8a0CMeGPxlLo5ooz3UI A7n1tCwKzPcZQSqbg2
CJSbFSXIqcGhdSRqIR add Q5NIUl9gu22TQL add c3yEdTN1s09I3GLPMhzeaO8tYO1oInH7z54GORoivPPH emj93xrriKpZBzfALmsZww1KjHWHF8SDGHyY8BDTQ
FtbegdsYgENmjNhLIfLfhBbaBNFyY34uIYCCW684Sn3mgpNLqurNckh 0FRRSXyEtQcmRhW00LrV6feqvsUiKge8H
Q5NIUl9gu22TQL display
APRvTmrEiNNTjKDBp2Fvw5szQaNg4gPrnPhgyCyrUyt9L6Y tMt
