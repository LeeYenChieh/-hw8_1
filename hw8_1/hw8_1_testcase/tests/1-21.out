XcXYeZrl5v4MGQWgza4A5pQWZa 
